tests/integration/test_validate_attribute.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
```

Comment:
This is the start of moving some files into an `tests/integration` directory. We can start slow and add new items into the appropriate locations while we slowly move items over in a strangler fig pattern: https://shopify.engineering/refactoring-legacy-code-strangler-fig-pattern

Reply:
I appreciate the additive nature of these changes

Reply:
Thanks Bryan! I like the change.